src/handlers/gestures/GestureDetector/Wrap.web.tsx

```diff
@@ -12,14 +12,23 @@ export const Wrap = forwardRef<HTMLDivElement, PropsWithChildren<{}>>(
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       const child: any = React.Children.only(children);
 
+      const isSVGElement =
+        Object.getPrototypeOf(child?.type)?.name === 'WebShape';
```

Comment:
Why this check is different from https://github.com/software-mansion/react-native-gesture-handler/pull/3197/files#diff-ec5692c69123b44d3a59cab1b401b52570a631f15fa37631163eb7502424fdbdR259-R265 ?

Reply:
Because those are 2 different things. Here we get React node:

<img width="699" alt="image" src="https://github.com/user-attachments/assets/4b8e927f-3e41-4d7e-a334-4b921cfdc920">

And in the line that you've linked we get ref to the actual `SVG` component (like `Circle`)

<img width="681" alt="image" src="https://github.com/user-attachments/assets/209d78ee-dcce-43d7-8de4-c7de2cdf7176">

Reply:
Can we rename one of those in this case?

Reply:
What about renaming the one in `Wrap.web.ts` into `isRNSVGComponent`? Or `isRNSVGNode`?

Reply:
Changed to `isRNSVGNode` in [73d7924](https://github.com/software-mansion/react-native-gesture-handler/pull/3197/commits/73d7924ee88c8aff768cdf4dbc36055801fddfc6)